include/golioth/location/wifi_zephyr.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <zephyr/net/wifi_mgmt.h>
+#include <golioth/location/wifi.h>
+
+static inline enum golioth_status golioth_location_wifi_append_zephyr(
```

Comment:
I believe this would be the first platform-specific public header in the SDK. Given the relatively minimal amount of functionality it offers, do you think it is necessary to include in the SDK, rather than just showing how this can be done in samples?

Reply:
Agree, that this is very minimal and can be used in samples directly.